Jenkinsfile.deploy

```diff
@@ -71,6 +71,21 @@ pipeline {
                 }
             }
         }
+        // TODO to remove
```

Comment:
This will be removed once test is done on node's modification.
See https://github.com/kiegroup/kogito-cloud-operator/pull/424/files#r450324260